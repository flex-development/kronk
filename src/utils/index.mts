/**
 * @file Entry Point - Utilities
 * @module kronk/utils
 */

export { default as isArgument } from '#utils/is-argument'
export { default as isCommand } from '#utils/is-command'
export { default as isCommandError } from '#utils/is-command-error'
export { default as isKronkError } from '#utils/is-kronk-error'
export { default as isOption } from '#utils/is-option'
