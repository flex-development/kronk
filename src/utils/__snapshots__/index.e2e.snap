// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`e2e:utils > should expose public api 1`] = `
[
  "isArgument",
  "isCommandError",
  "isKronkError",
  "isKronkEvent",
  "isOption",
]
`;
